ktlint/src/main/kotlin/com/pinterest/ktlint/internal/FileUtils.kt

```diff
@@ -163,7 +162,6 @@ internal fun File.location(
 /**
  * Run lint over common kotlin file or kotlin script file.
  */
-@OptIn(FeatureInAlphaState::class)
```

Comment:
Cool that you removed those. I still had that on my list. I found some more, which I have removed and added to this PR.